exercises/E-conditionals/exercise.js

```diff
@@ -9,6 +9,10 @@
 var name = "Daniel";
 var danielsRole = "mentor";
 
+if (danielsRole = "mentor") {
+  console.log("Hi, I'm Daniel, I'm a mentor.")
```

Comment:
Hey just a heads-up, it's best practice to not hardcode the work, rather replace the words `mentor` and `Daniel` with it's appropriate declared variables.

Reply:
Other than that, the rest of your code looks great and I know you've probably had a busy week, but just make sure that the mandatory folder is completed as well as it is due by the end of today.